website/docs/reference/change-requests.mdx

```diff
@@ -10,61 +10,50 @@ import VideoContent from '@site/src/components/VideoContent.jsx';
-
-* **Draft** - The change request is in draft mode, and can be edited by the user that created the draft.
-* **In review** - The change request is in review mode, and can be edited by the user that created the draft. If editing occurs, all current approvals are revoked
-* **Approved** - The change request has been approved by the required number of users.
-* **Scheduled** - The change request has been scheduled and will be applied at the scheduled time (unless there are conflicts, as described in the section on [scheduling change requests](#scheduled-changes)).
-* **Applied** - The change request has been applied to the environment. The feature flag configuration is updated.
-* **Cancelled** - The change request has been cancelled by the change request author or by an admin.
-* **Rejected** - The change request has been rejected by the reviewer or by an admin.
+Once you make the first modification in an environment that has change requests enabled, any subsequent changes must be added to the same draft. This draft remains private to its author until it's submitted for review. While a draft is active, a banner at the top of the screen informs the author that changes are pending.
 
-![Change request banner](/img/change-request-banner.png)
+The lifecycle of a change request includes:
+* **Draft** - The change request is in draft mode, and can be edited by the author.
+* **In review** - The request has been submitted for review. The author can still make edits, but doing so will revoke any existing approvals.
+* **Approved** - The request has received the required number of approvals.
+* **Scheduled** - The request is scheduled to be applied at a future time (assuming no [conflicts](#conflicts)).
+* **Applied** - The changes have been successfully applied to the environment.
+* **Cancelled** - The change request has been cancelled by the author or by an admin.
+* **Rejected** - The change request has been rejected by a reviewer or by an admin.
 
-Once a change request is sent to review by the user who created it, it becomes available for everyone in the change request tab in the project.
+Once submitted, the change request appears in the project's **Change requests** tab. From there, you can view details of the proposed changes, the current status of the request, and the next steps required. Users with sufficient persmissions can approve or reject, and apply or schedule the changes.
 
-From here, you can navigate to the change request overview page. This page will give you information about the changes the change request contains, the state the change request is in, and what action needs to be taken next.
+![Change request overview](/img/change-request-overview.png)
 
-![Change request banner](/img/change-request-overview.png)
-
-From here, if you have the correct permissions, you can approve and schedule or apply the change request. Once applied, the changes will be live in production.
-
-### Scheduled changes
+### Schedule a change
```

Comment:
Did you consider "Scheduled change request"? Would that be more precise on what this capability actually does?